src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -33,7 +33,6 @@ import { Schemas } from 'vs/base/common/network';
 import { ILogService } from 'vs/platform/log/common/log';
 import { toErrorMessage } from 'vs/base/common/errorMessage';
 import { NotebookChangeType } from 'sql/workbench/services/notebook/common/contracts';
-import { find, firstIndex } from 'vs/base/common/arrays';
```

Comment:
Both these apis are deprecated by vscode guys replacing its usage with their suggesion of Array.find and Array.findIndex for these.